Controller/Process/Json.php

```diff
@@ -252,16 +252,15 @@ private function processNotification(array $response, $notificationMode)
         }
 
         // Validate if Ip check is enabled and if the notification comes from a verified IP
-        if ($this->configHelper->getNotificationsIpCheck() && !$this->isIpValid()) {
+        if (!$this->isIpValid()) {
```

Comment:
Let's remember to mention in the release notes that the IP and HMAC checks will always be on